app/models/CrystallisationStatus.scala

```diff
@@ -51,7 +50,7 @@ case class CrystallisationStatus(status: String,
       Json.obj("calculationId" -> "1d35cfe4-cd23-22b2-b074-fae6052024a8",
         "calculationTimestamp" -> s"2024-09-30T09:15:34.0Z",
         "calculationType" -> calculationTypeField) ++
-        ("taxYear", taxYearField) ++
```

Comment:
I used the ++ syntax with the Option to eliminate the field when the value of the option is None (if tax year is 2024 or after). Does
`Json.obj("taxYear" -> taxYearField)`
do the same?

Reply:
reverted